src/render/decorations/CHyprGroupBarDecoration.hpp

```diff
@@ -35,11 +35,15 @@ class CHyprGroupBarDecoration : public IHyprWindowDecoration {
 
     virtual void                       damageEntire();
 
-    virtual void                       onBeginWindowDragOnDeco(const Vector2D&);
+    virtual bool                       onBeginWindowDragOnDeco(const Vector2D&);
 
-    virtual bool                       onEndWindowDragOnDeco(CWindow* pDraggedWindow, const Vector2D&);
+    virtual bool                       onEndWindowDragOnDeco(const Vector2D&, CWindow*);
 
-    virtual void                       onMouseButtonOnDeco(const Vector2D&, wlr_pointer_button_event*);
+    virtual bool                       onMouseButtonOnDeco(const Vector2D&, wlr_pointer_button_event*);
+
+    virtual bool                       onScrollOnDeco(const Vector2D&, wlr_pointer_axis_event*);
```

Comment:
these no longer need to be virtual; they do not override anything. They can also go private.

Reply:
Done.

Do any of those need to be marked as virtual, since as far as I understand virtual is inherited? Or you're just being explicit?

Reply:
no point in these being virtual as nothing will inherit groupbar

Reply:
No, I meant "those". "Those" as in any of those other public methods (`getDecorationType`, `updateWindow`...).